nima/http2/http2/src/main/java/io/helidon/nima/http2/Http2Headers.java

```diff
@@ -782,9 +782,8 @@ enum StaticHeader implements IndexedHeaderRecord {
                 if (predefinedHeader.hasValue()) {
                     BY_NAME_VALUE.computeIfAbsent(predefinedHeader.headerName().lowerCase(), it -> new HashMap<>())
                             .put(predefinedHeader.value(), predefinedHeader);
-                } else {
-                    BY_NAME_NO_VALUE.put(predefinedHeader.headerName().lowerCase(), predefinedHeader);
                 }
+                BY_NAME_NO_VALUE.putIfAbsent(predefinedHeader.headerName().lowerCase(), predefinedHeader);
```

Comment:
maybe comment why? Such as "indexed headers may be referenced either with or without value, so we need to store them in both tables"